src/set_theory/game/birthday.lean

```diff
@@ -134,7 +130,46 @@ theorem le_birthday : ∀ x : pgame, x ≤ x.birthday.to_pgame
 le_def.2 ⟨λ i, or.inl ⟨to_left_moves_to_pgame ⟨_, birthday_move_left_lt i⟩,
   by simp [le_birthday (xL i)]⟩, is_empty_elim⟩
 
-theorem neg_birthday_le (x : pgame) : -x.birthday.to_pgame ≤ x :=
+variables (a b x : pgame.{u})
+
+theorem neg_birthday_le : -x.birthday.to_pgame ≤ x :=
 let h := le_birthday (-x) in by rwa [neg_birthday, ←neg_le_iff, neg_neg] at h
 
+@[simp] theorem birthday_add : ∀ x y : pgame.{u}, (x + y).birthday = x.birthday ♯ y.birthday
+| ⟨xl, xr, xL, xR⟩ ⟨yl, yr, yL, yR⟩ := begin
+  rw [birthday_def, nadd_def],
+  simp only [birthday_add, lsub_sum, mk_add_move_left_inl, move_left_mk, mk_add_move_left_inr,
+    mk_add_move_right_inl, move_right_mk, mk_add_move_right_inr],
+  rw [max_assoc, ←max_assoc (lsub (λ b : yl, _)), max_comm (lsub (λ b : yl, _)), max_assoc,
+    ←max_assoc],
+  congr; apply le_antisymm,
+  any_goals
+  { exact max_le_iff.2 ⟨lsub_le_iff.2 (λ i, lt_blsub _ _ (birthday_move_left_lt i)),
+      lsub_le_iff.2 (λ i, lt_blsub _ _ (birthday_move_right_lt i))⟩ },
+  all_goals
+  { apply blsub_le_iff.2 (λ i hi, _),
+    rcases lt_birthday_iff.1 hi with ⟨j, hj⟩ | ⟨j, hj⟩ },
+  { exact lt_max_of_lt_left ((nadd_le_nadd_right hj _).trans_lt (lt_lsub _ _)) },
+  { exact lt_max_of_lt_right ((nadd_le_nadd_right hj _).trans_lt (lt_lsub _ _)) },
+  { exact lt_max_of_lt_left ((nadd_le_nadd_left hj _).trans_lt (lt_lsub _ _)) },
+  { exact lt_max_of_lt_right ((nadd_le_nadd_left hj _).trans_lt (lt_lsub _ _)) }
+end
+using_well_founded { dec_tac := pgame_wf_tac }
+
+theorem birthday_add_zero : (a + 0).birthday = a.birthday := by simp
+theorem birthday_zero_add : (0 + a).birthday = a.birthday := by simp
+theorem birthday_add_one  : (a + 1).birthday = a.birthday + 1 := by simp
+theorem birthday_one_add  : (1 + a).birthday = a.birthday + 1 := by simp
+
+@[simp] theorem birthday_nat (n : ℕ) : birthday n = n :=
```

Comment:
```suggestion
@[simp] theorem birthday_nat_cast (n : ℕ) : birthday n = n :=
```
matches the naming elsewhere in mathlib

Reply:
Should `birthday_add_nat` and `birthday_nat_add` be renamed too?